src/components/homepage/Homepage.jsx

```diff
@@ -27,9 +27,23 @@ const Homepage = () => {
   const grants = useSelector((state) => {
     return state.homePage.grantsInfo;
   });
+  const state = useSelector((state) => {
+    return state;
+  });
   const user = useSelector((state) => {
     return state.homePage.userInfo;
   });
+  const testUser = useSelector((state) => {
```

Comment:
I can't seem to figure out where this testUser is being used, could you help clear it up for me?

Reply:
Short answer: it's not doing anything.....yet O.o 
Currently user is being set with hard coded data, but testuser is trying to set the user based on the state that's returned when the user logs in.